### DISCUSSION TITLE
Hubs Cloud Upgrade new version in Digital Ocean

### DISCUSSION THREAD

**Discussion by arvisionlab** [mozilla/hubs#3176] [Hubs Cloud Questions] [answered]
I have installed hubs cloud in DO, how to upgrade/update the new version after installation?

**Comment by netpro2k** [comment, marked as answer]
Updates to all the various services that make up Hubs are applied automatically through Chef Habitat channels. When your server first starts up it will pull down the latest updates, and will keep itself up to date. They only time you would need to manually apply updates is if you are running a custom client.
